contracts/cw3-flex-multisig/src/contract.rs

```diff
@@ -192,15 +193,33 @@ pub fn execute_execute(
     info: MessageInfo,
     proposal_id: u64,
 ) -> Result<Response, ContractError> {
-    // anyone can trigger this if the vote passed
-
     let mut prop = PROPOSALS.load(deps.storage, proposal_id)?;
     // we allow execution even after the proposal "expiration" as long as all vote come in before
     // that point. If it was approved on time, it can be executed any time.
     if prop.current_status(&env.block) != Status::Passed {
         return Err(ContractError::WrongExecuteStatus {});
     }
 
+    // Executor can be set in 3 ways:
+    // - Member: any member of the voting group can execute
+    // - Only: only passed address is able to execute
+    // - None: Anyone can execute message
+    let cfg = CONFIG.load(deps.storage)?;
+    if let Some(executor) = cfg.executor {
+        match executor {
```

Comment:
Is this something that could be pulled out to a method on Executor?
Already thinking of reusing it for who can make proposals. (but default is Member)

Reply:
I moved it under `Config` struct. I think it's more versatile this way.
What do you think?